searx/webapp.py

```diff
@@ -584,6 +584,11 @@ def index():
     )
 
 
+@app.route('/health', methods=['GET'])
```

Comment:
After a quick research, it seems to me that the convention is to use `/healthz` as the name of the endpoint. Is that correct? Do you mind changing it?

Reply:
From my experience - ever rudo y use different one - I’ve seen `/ping` as well. So, no worries - healthz is fine as well :)

Reply:
Done 👍